### PR TITLE
[12.0] [MIG] account_move_batch_validate

### PR DESCRIPTION
**Pull request by i-vyshnevska**

### CODE REVIEW

File: account_move_batch_validate/views/account_move.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+<odoo>
+
+    <record model="ir.ui.view" id="account_move_form">
+        <field name="name">account.move.form in (account_move_batch_validate)</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form"/>
+        <field name="arch" type="xml">
+
+            <field name="ref" position="after">
+                <field name="to_post"/>
```
Comment (i-vyshnevska) [inline account_move_batch_validate/views/account_move.xml:15]: this field is technical and readonly, seems this view is redundant

File: account_move_batch_validate/wizard/account_move_validate.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        required=True, default='mark')
+    eta = fields.Integer(string="Seconds to wait before starting the jobs")
+    asynchronous = fields.Boolean(string="Use asynchronous validation")
+
+    @api.model
+    def _get_actions(self):
+        return [
+            ('mark', 'Mark for posting'),
+            ('unmark', 'Unmark for posting')
+        ]
```
Comment (jcoux) [inline account_move_batch_validate/wizard/account_move_validate.py:38]: Actions are not translatable like this. Move them directly on field definition (and remove function).

File: account_move_batch_validate/readme/CONTRIBUTORS.rst [hunk 3]
```diff
@@ -0,0 +1,7 @@
+
```
Comment (jcoux) [inline account_move_batch_validate/readme/CONTRIBUTORS.rst]:
```suggestion
```

File: account_move_batch_validate/readme/USAGE.rst [hunk 4]
```diff
@@ -0,0 +1,24 @@
+This module provides a wizard to post many Journal Entries in batch. it
```
Comment (jcoux) [inline account_move_batch_validate/readme/USAGE.rst]: This file is a duplicate of DESCRIPTION.rst

**Review by jcoux — CHANGES_REQUESTED** [review]

File: account_move_batch_validate/wizard/account_move_validate.py (hunk 2 above)
Comment (i-vyshnevska) [reply account_move_batch_validate/wizard/account_move_validate.py:38]: thanks, I think explicit marking them as translatable will solve the issue

**Review by jcoux — APPROVED** [review]
Technically seems good

**Review by tschanzt — APPROVED** [review]

File: account_move_batch_validate/wizard/account_move_validate.py [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+            return super().validate_move()
+
+        wizard_data = {
+            'move_ids': self.env.context.get('active_ids'),
+            'action': self.action,
+            'asynchronous': self.asynchronous,
+            'eta': self.eta,
+        }
+
+        if self.env.context.get('automated_test_execute_now'):
```
Comment (grindtildeath) [inline account_move_batch_validate/wizard/account_move_validate.py:54]:
```suggestion
        if odoo.tools.config['test_enable'] and self.env.context.get('automated_test_execute_now'):
```

**Review by grindtildeath — APPROVED** [review]
LGTM :+1:

**Review by TDu — APPROVED** [review]

File: account_move_batch_validate/models/account_move.py [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+
+    to_post = fields.Boolean(
+        string="Posting requested", readonly=True,
+        help="Check this box to mark the move for batch posting")
+    post_job_uuid = fields.Char(string="UUID of the Job to approve this move")
+
+    @api.multi
+    @job(default_channel='root.account_move_batch_validate')
+    def validate_one_move(self):
+        if self.exists():
```
Comment (sebalix) [inline account_move_batch_validate/models/account_move.py:37]: To be correct, either this method should call `ensure_one()` or get the result of `self.exists()` and call `post()` on it. Right now we could call this method on a len(recordset) > 1 with a record filtered by `exists()`.

**Review by sebalix — APPROVED** [review]
Small remark, but :+1:

**Pull request merged**